pkg/karmadactl/cmdinit/utils/examples.go

```diff
@@ -40,6 +44,12 @@ metadata:
   name: karmada-agent-sa
   namespace: karmada-system
 ---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: karmada-impersonator
+  namespace: karmada-cluster
+---
```

Comment:
The `karmada-impersonator` now created by `karmada-agent` at startup phase now, by #1170. 
So, we don't have to create it here anymore.